### PR TITLE
add changes in order to build on windows os

### PR DESCRIPTION
**Pull request by marahmbawwab**

### CODE REVIEW

**Comment by fundamental** [comment]
Hello Marah,

Thanks for the contribution and I'm glad to see that hotloading works on windows with minimal changes.

Would you mind updating the commit message to something more informative? In this project we roughly conform to the git message conventions documented in https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html .

In this case something along the lines of:

```
Add build instructions for windows hotloading

- Add windows-hotload target to Makefile
- Mention new target in contributing documentation
```

You should be able to update your commit by changing it locally via `git commit --amend` and then update your fork via a `git push --force` command. 

Also a minor note, but generally creating a branch in your fork to create pull requests for is a best practice. It's never strictly necessary, but it does avoid headaches when multiple PRs are occurring, or if your repo's history otherwise gets out of sync with the upstream repository.

File: contributing.adoc
Comment (pgervais) [inline contributing.adoc:33]: This section is meant to contain only code, which means that most people will expect being able to directly copy-paste the content and use it. What you added at the end of the lines (e.g. "for linux") makes that impossible.

You have at least two options here:
- make things valid by adding a proper comment; `make linux-hotload  # for linux`.
- split the block into two. Something like

```
For Linux:
--------------------------------------------------------------------------------
make linux-hotload
--------------------------------------------------------------------------------

For Windows:
--------------------------------------------------------------------------------
make windows-hotload
--------------------------------------------------------------------------------
```

**Review by pgervais — CHANGES_REQUESTED** [review]

**Comment by pgervais** [comment]
This change looks good to me now, thanks for taking into account all the comments. I can spend some time with you later to give more details about this "not working with the master branch suggestion".

I'll let @fundamental finish the review since I don't have merge permissions on this repo.

**Comment by fundamental** [comment]
Thanks for the contribution :+1:

**Pull request merged**
